scalafmt-core/shared/src/main/scala/org/scalafmt/config/SortSettings.scala

```diff
@@ -0,0 +1,53 @@
+package org.scalafmt.config
+
+import metaconfig._
+
+@DeriveConfDecoder
```

Comment:
Best to replace this with a hand-written decoder that validates that all modifiers are defined, and produces a helpful error message saying which modifier is missing in case something is wrong.

Reply:
Nevermind this comment.